### PR TITLE
Re-arrange form inputs, add postal code to person struct

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

- Moves position of travel and crypto address inputs.
- Adds postal code and notes to form.
- Adds postal code to `Person` struct.

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [x] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/26646962?key=fd956b7614841eb312fd4ed87be7dfeb

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/api/v1/prepared.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type Person struct {
 	AddrLine2     string `json:"addr_line_2"`
 	City          string `json:"city"`
 	State         string `json:"state"`
+	PostalCode    string `json:"post_code"`
```
Comment (daniellemaxwell) [inline pkg/web/api/v1/prepared.go:33]: I looked at the ivms101 proto file and believe this is correct.

Comment (bbengfort) [reply pkg/web/api/v1/prepared.go:33]: Yep - that works!

File: pkg/web/api/v1/prepared.go [hunk 2]
```diff
@@ -72,7 +73,7 @@ func (p *Person) NaturalPerson() *ivms101.Person {
 	addrLines := []string{
 		strings.TrimSpace(p.AddrLine1),
 		strings.TrimSpace(p.AddrLine2),
-		strings.TrimSpace(fmt.Sprintf("%s, %s", p.City, p.State)),
+		strings.TrimSpace(fmt.Sprintf("%s, %s, %s", p.City, p.State, p.PostalCode)),
```
Comment (bbengfort) [inline pkg/web/api/v1/prepared.go:76]: Thanks for adding it here!

**Review by bbengfort — APPROVED** [review]
Good work!

**Pull request merged**